# CustomMsg.msg
Header header
string data
